custom_components/powercalc/data/signify/LTP002/model.json

```
@@ -0,0 +1,15 @@
+{
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly Plug S",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v0.26.1:docker"
+    },
+    "name": "Hue ambiance pendant (LTP002)",
```

Mentioning the model number again would be a bit verbose and not consistent with the other lights in supported models list, so that may be removed.

```suggestion
    "name": "Hue ambiance pendant",
```